### ISSUE TITLE
emb-w-03 - Processing Files Dialog

### ISSUE THREAD

**Issue by bertramlyons** [LibraryOfCongress/embARC#4]
emb-w-03	Processing Files Dialog	Yes	1-Critical	

JAWS does not announce the dialog contents; NVDA announces it, but only once with no ability to review the information.	Ensure that dialog contents are accessible to screen readers and can be reviewed as necessary.

**Comment by bertramlyons** [comment]
In process.

**Comment by bertramlyons** [comment]
Ticket complete - to be reviewed.

**Comment by bertramlyons** [comment]
Completed and merged in version 1.3.0.
